nuitka/Options.py

```diff
@@ -354,15 +354,15 @@ def shallClearPythonPathEnvironment():
 
 
 def shallUseStaticLibPython():
-    """ *bool* = derived from sys.version
+    """ *bool* = derived from `sys.prefix` and `os.name`
 
     Notes:
         Currently only AnaConda on non-Windows can do this.
     """
 
     # For AnaConda default to trying static lib python library, which
     # normally is just not available or if it is even unusable.
-    return "Anaconda" in sys.version and os.name == "nt"
+    return os.path.exists(os.path.join(sys.prefix, 'conda-meta')) and not os.name == "nt"
```

Comment:
Yes, that is clearly much better, however did a `not` snuggle in there in that condition? I assume, you want to do it not just for Windows, which is what this code did. I therefore think it might break Windows.

I was defensive about this, as static Python didn't work unless you used the precisely same gcc version than Anaconda did. I was e.g. unlucky with gcc-6 and gcc-8 on Linux, where it is compiled with gcc-7, one was saying too new bytecode, the other was crashing when using it to link with cryptic errors.

Reply:
Maybe I missed something: as far as I understood, static linking is not feasible in Windows OS, so the `not os.name == "nt"` condition...

Reply:
You are right, I need to check how this came to be, the doc comment also states that. You just correct a mistake there, great. With this error, the feature was disabled. Let me take the time to check the history there before getting back to this.